### ISSUE TITLE
0 out of 0 files sent

### ISSUE THREAD

**Issue by HexCodeFFF** [thura10/airwave#5]
Hello, I'm trying to send a png file to my phone ([specifically this one](https://cdn.discordapp.com/attachments/235888301930708992/683596138821189632/unknown.png)), but i am coming across an issue.
I clicked send files on desktop, dragged the png over, and tapped receive files on my phone. My desktop shows "<phone name>'s iPhone 0 out of 0 files sent."
![image](https://user-images.githubusercontent.com/35177464/75622707-feeea300-5b68-11ea-8330-7de4f67d23dd.png)
I am not sure what is causing this and I can try other things to figure out what's going on. I am able to send files from my phone to my pc just fine.

**Comment by thura10** [comment]
Hi. Are you able to send other files or other types of files? Is there any special character in the file name? Can you try selecting the file instead of dragging over?
If all else fails, press Ctrl+Shift+I to open developer console, Go to Console tab and see if there are any error messages. Thank you.

**Comment by HexCodeFFF** [comment]
Ok, I found sort of what's going on. The console displays nothing. The issue is dragging and dropping images from Chrome. Airwave goes to the next screen as if i've added a file to send, but it doesn't register.
[airwavebugmp4.zip](https://github.com/thura10/airwave/files/4302513/airwavebugmp4.zip)

**Comment by HexCodeFFF** [comment]
Saving the file and dragging it over shows the file name but still shows "0 out of 0 files sent" (even from another pc). Using the file selection tool also has the same behavior. 
Here is my test file
![test](https://user-images.githubusercontent.com/35177464/76152549-58cdfc00-6086-11ea-877d-d097297b56d9.png)

**Comment by thura10** [comment]
Ok. There's two parts to this error from what I can see. When you drag an image from a browser, it doesn't really drop the file but instead drop a link to the image. So there is actually no file in the list when you try to send it. I was able to replicate this and have fixed it. (Well stop the app from going to the other screen. Can't drag and drop links yet.)

But when I download the file and send it, the file got to my phone fine. I did have the same issue from someone else when the file name had special characters. Are you on the latest version?
